src/pages/_app.tsx

```diff
@@ -43,7 +81,10 @@ export default function App({ Component, pageProps }: AppProps) {
   return (
     <>
       <Head />
-      <Component {...pageProps} />
+      {/* this wrapper is required for the next/font loaded fonts to work with tailwind */}
+      <div className={`${marianne.variable} font-sans`}>
+        <Component {...pageProps} />
+      </div>
```

Comment:
brainsto: Could we encapsulate the font logic in a dedicated file/components, which would expose a `FontProvider` or something else?